ArrayTests/GlobalSuppressions.cs

```diff
@@ -0,0 +1,36 @@
+﻿// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Naming", "CA1707:Identifiers should not contain underscores", Justification = "Test Methods Should Contain Underscores", Scope = "member", Target = "~M:Assignment6.ArrayTests.ArrayString_SetGetInBounds_ReturnsCorrectValue")]
```

Comment:
If you get rid of the target and scope tags the suppression will work for the entire ArrayTests project, I know how annoying it is to individually suppress all of those!
```suggestion
[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Naming", "CA1707:Identifiers should not contain underscores", Justification = "Test Methods Should Contain Underscores")]
```